b2sdk/utils.py

```diff
@@ -366,6 +366,24 @@ class B2TraceMetaAbstract(DefaultTraceAbstractMeta):
     pass
 
 
+class ConcurrentUsedAuthTokenGuard(object):
```

Comment:
It's nice to have a doc string for each class, with enough information to know how to use the class without reading the code.

Reply:
@bwbeach addressed in https://github.com/Backblaze/b2-sdk-python/pull/89/commits/7ec6e2589bdb48dbeda0661335727b502c3943bc